packages/lesswrong/components/common/LWHome.tsx

```diff
@@ -32,21 +32,21 @@ const LWHome = () => {
           </SingleColumnSection>}
           
           <AnalyticsInViewTracker
-              eventProps={{inViewType: "homePosts"}}
-              observerProps={{threshold:[0, 0.5, 1]}}
+            eventProps={{inViewType: "homePosts"}}
+            observerProps={{threshold:[0, 0.5, 1]}}
           >
-            <LWHomePosts />
+            <LWHomePosts>
+              <QuickTakesSection />
+    
+              <EAPopularCommentsSection />
+    
+              <RecentDiscussionFeed
+                af={false}
+                commentsLimit={4}
+                maxAgeHours={18}
+              />
```

Comment:
Hm... does making these children of `LWHomePosts` have implications for what analytics events get recorded due to the parent `AnalyticsInViewTracker`?

Reply:
Theoretically but in practice EAPopularComments and RecentDiscussionFeed will overwrite the LWHomePosts pageSectionContext with their own. QuickTakes doesn't have one of its own but I'll add one now